src/reaction_network.jl

```diff
@@ -123,7 +123,11 @@ const forbidden_variables_error = let
 end
 
 # Declares the keys used for various options.
+<<<<<<< Updated upstream
 const option_keys = (:species, :parameters, :variables, :ivs)
+=======
+const option_keys = (:species, :parameters, :variables, :ivs, :discrete_events)
+>>>>>>> Stashed changes
 
```

Comment:
This looks like something has gotten messed up when trying to merge master into your PR. There should be such comments listed.